subjects/flags/README.md

```diff
@@ -9,6 +9,10 @@ This program should :
 - Insert the string given to the `--insert` (or `-i`), into the argument `string`, if given.
 - If the flag `--order` (or `-o`) is given, order the `string` argument (in ASCII order).
 - If there are no arguments or if the flag `--help` (or `-h`) is given, the options should be printed as in the example.
+  - The short flag will have two spaces before the (-).
+  - The explanation of the flag will have a tab followed by a space before the beginning of the sentence (This flag...).
+
+***Note:*** Don't mind the extra spaces or tabs on the following example as they were placed there to provide a better understanding and visualization of the output in the terminal. Follow the rules above for the spacing.
```

Comment:
We could use a quote for this warning `>`. That is just a suggestion.

Reply:
i changed it to a warning, thanks for the tip.